acmwebsite/model/banner.py

```diff
@@ -10,4 +10,4 @@ class Banner(DeclarativeBase):
 
     id = Column(Integer, autoincrement=True, primary_key=True)
     photo = Column(UploadedFileField(upload_type=UploadedImageWithThumb))
-    description = Column(Unicode(255), unique=True)
+    description = Column(Text, unique=True)
```

Comment:
Per https://github.com/ColoradoSchoolOfMines/acm-website/pull/32#discussion_r154513923, I think we want to still use `Unicode`, but just with like a 2048 char limit.

Deferring to @jackrosenthal on this.